app/src/main/kotlin/com/intive/picover/main/theme/Theme.kt

```diff
@@ -54,9 +60,14 @@ fun PicoverTheme(
 			}
 		}
 	}
-	MaterialTheme(
-		colorScheme = colorScheme,
-		typography = Typography,
-		content = content,
-	)
+	val imageComponent = imageComponent {
+		+PlaceholderPlugin.Failure(painterResource(R.drawable.ic_avatar_placeholder))
```

Comment:
Could it be also used for loading state?

Reply:
Yes, it can - https://github.com/skydoves/landscapist#placeholderplugin